Zotero/Scenes/Detail/PDF/ViewModels/PDFReaderActionHandler.swift

```diff
@@ -1547,7 +1547,8 @@ final class PDFReaderActionHandler: ViewModelActionHandler, BackgroundDbProcessi
                             trimmedRects.append(currentRect)
                             continue
                         }
-                        let trim = intersection.height / 2
+                        // Each rect is trimmed by half the intersection height, plus 0.25 to have a small gap between the lines.
+                        let trim = (intersection.height / 2) + 0.25
```

Comment:
I'm not against this hard coded value, but does it work for all/most documents? What about different font sizes or such?

Reply:
It's in the annotation's normalized PDF coordinates space, so it shouldn't be affected by font sizes. The goal is to have just a small gap, so that rects do not be viewed as a seamless block, but probably we don't want it to be in relation to the rects' height, as a larger part of the text glyphs ascenders/descenders may get out of the rect. In any case, I'll test with more documents first.

Reply:
Fixed gap size works well for different font spaces. 

While testing, I found some edge cases, when the whole page is selected, due to the way PSPDFKit sorts the selection rects. Specificallly, it might place a rect on top, especially if it is centered on page, between two consecutive line rects that are on the page footer, resulting on skipping an overlap. Since this is a contrived selection to annotate, I'd argue that we shouldn't concern ourselves for now, unless we have reports from users.

Reply:
Sounds good, feel free to merge.